src/ros/ros2/ros2.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Andrew Short. All rights reserved.
```

Comment:
use Microsoft copyright header for new code?

Reply:
Good catch. Fixed it.